groups/staffware-db/profiles/heritage-live-eu-west-2/vars

```diff
@@ -74,3 +74,12 @@ failover_approvers = [
 ]
 
 netapp_ips = ["10.84.9.97"]
+
+# CHIPS DB SGs
+chips_db_sg = [
```

Comment:
chips_db_sg -> staffware_db_sg, same for the comment

Reply:
Changed to staffware_db_sg